### PR TITLE
Add ops coordinator and ops manager roles

### PR DESCRIPTION
**Pull request by chrisblackburn**

### CODE REVIEW

File: roles/as_code/lib/operations_coordinator.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require_relative './shared_expectations/operations_coordinator_expectations'
+
+JobSpec::Role.definition 'Operations Coordinator' do
+  description <<~DESCRIPTION
+    Our Operations Coordinator supports the smooth running of Made Tech by ensuring deliveries are well resourced, that the Live Services team is well supported, and through reporting on billing and utilisation.
```
Comment (EThom73) [inline roles/as_code/lib/operations_coordinator.rb]: Remove Live Services responsibilities

File: roles/as_code/lib/operations_manager.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  expected 'to proactively identify significant improvements to business processes',
+    'We expect our Operations Manager to proactively identify and implement major improvements to how Made Tech operates, such as identifying the need to improve our credit control processes.'
+
+   expected 'to develop roadmaps to deliver strategic business objectives',
+   	'We expect our Operations Manager to understand quarterly business objectives, to develop roadmaps, and to prioritise their workload to ensure these objectives are delivered to the desired timescale.'
+
+   expected 'to implement and monitor policies to ensure the security of company and customer data',
+     'We expect our Operations Manager to implement policies and practices to ensure Made Tech is compliant with any relevant information security accreditations and legal requirements.'
+
+   expected 'to proactively analyse performance metrics and propose and implement changes to improve key metrics',
```
Comment (EThom73) [inline roles/as_code/lib/operations_manager.rb]: Interested to know what these are for Ops. Beyond Company Scorecard, do we have any?

File: roles/as_code/lib/shared_expectations/operations_coordinator_expectations.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+class OperationsCoordinatorExpectations < JobSpec::Role::Expectations
+ expected 'to report on and maintain team utilisation at > 85%',
+    'We expect the Operations Coordinator to report on utilisation and maintain a schedule that supports 85% of the billable team capacity at Made Tech to be utilised on a weekly basis.'
+
+  expected 'to control resource allocation and forecasting across the organisation',
+    'we expect the Operations Coordinator to appropriately resource teams to ensure both productive and appropriately staffed teams, and individual happiness. It is expected that resource forecasting will be accurately maintained, including upcoming holidays, to allow forecasted budgets to be reported to customers.'
```
[the remark this answers is not in the document]
Comment (chrisblackburn) [reply roles/as_code/lib/shared_expectations/operations_coordinator_expectations.rb]: I think it's a role responsibility to ensure we're making these decisions with awareness of things such as people's aspirations. I don't think this prescribes _how_ that knowledge is obtained.

Does that match with your view?

[the remark this answers is not in the document]
Comment (EThom73) [reply roles/as_code/lib/shared_expectations/operations_coordinator_expectations.rb]: Sounds fine when framed that way

**Pull request merged**